config/samples/layout/multibackend/multibackend.yaml

```diff
@@ -0,0 +1,55 @@
+apiVersion: manila.openstack.org/v1beta1
+kind: Manila
+metadata:
+  name: manila
+  namespace: openstack
+spec:
+  manilaAPI:
+      customServiceConfig: |
+         [DEFAULT]
+         enabled_share_protocols=nfs,cephfs,cifs
+  manilaShares:
+    share0:
+      customServiceConfig: |
+         [DEFAULT]
+         enabled_share_backends = cephfs
+         [cephfs]
+         driver_handles_share_servers=False
+         share_backend_name=cephfs
+         share_driver=manila.share.drivers.cephfs.driver.CephFSDriver
+         cephfs_conf_path=/etc/ceph/ceph.conf
+         cephfs_auth_id=openstack
+         cephfs_cluster_name=ceph
+         cephfs_protocol_helper_type=CEPHFS
+      replicas: 1
+    share1:
+      customServiceConfig: |
+          [DEFAULT]
+          enabled_share_backends=cephfsnfs
```

Comment:
this is pretty cool; i didn't think to enable this in the kuttl tests - we _dont_ actually check if the nfs cluster exists on the backend during driver initialization; which means that we won't see errors until we create shares. 8)

Reply:
Should `cephfsnfs` have a comma in it, i.e. `cephfs,nfs`?

Reply:
hey @abays, no because `cephfsnfs` refers to the section on L29 that points to the `NFS` driver